pyproject.toml

```diff
@@ -208,14 +208,14 @@ enabled = true
 [tool.gilesbot.towncrier_changelog]
 enabled = true
 changelog_skip_label = "No changelog entry needed"
-help_url = "https://github.com/PlasmaPy/PlasmaPy/blob/main/changelog/README.rst"
-changelog_missing = "Missing changelog entry"
-changelog_missing_long = "This pull request needs a changelog entry file in `changelog/NUMBER.TYPE.rst`. For more information, consult https://github.com/PlasmaPy/PlasmaPy/blob/main/changelog/README.rst "
+help_url = "https://docs.plasmapy.org/en/latest/contributing/changelog_guide.html#adding-a-changelog-entry"
+changelog_missing = "Missing changelog entry (see `changelog/README.rst`)"
+changelog_missing_long = "This pull request needs a changelog entry at `changelog/NUMBER.TYPE.rst`, where `NUMBER` is the pull request number and `TYPE` is one of `feature`, `trivial`, `doc`, `bugfix`, `breaking`, or `removal`.\n\nFor details, see: https://docs.plasmapy.org/en/latest/contributing/changelog_guide.html#adding-a-changelog-entry"
 verify_pr_number = true
-number_incorrect = "Incorrect changelog entry number (match PR!)"
-number_incorrect_long = "The changelog entry's number does not match this pull request's number."
-type_incorrect = "Incorrect changelog entry type (see list in changelog README)"
-type_incorrect_long = "The changelog entry for this PR must have one of the types (as in NUMBER.TYPE.rst) as described in the changelog README)."
+number_incorrect = "Changelog entry number ≠ PR number (see `changelog/README.rst`)"
+number_incorrect_long = "The changelog entry number does not match this pull request's number.\n\nWhen purposefully editing the changelog entry for a different pull request, set the 'No changelog entry needed' label to ignore this check."
```

Comment:
After this is merged, I'll have to check to make sure that the `\n` gets interpreted as a line break.